src/content/docs/products/splashkit/SplashKit Tutorials/Onboarding/02-linux-installation.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Linux Installation Guide
+description: Use this template to create a proposal for a new SplashKit tutorial.
+sidebar:
+  order: 2
+  label: 2. Linux Installation
+  attrs:
+    class: linux
+---
+
+This is a guide on how to install Splashkit on Ubuntu, along with all the necessary dependencies and software required to run Splashkit. This guide is intended to get you set up and ready to contribute to the SplashKit team if you're a Linux user. This installation guide was done on Ubuntu 24.04 LTS.
+
+## Installing Necessary Packages
+
+### Update your system
+
+To start, open up a terminal with the keys `ctrl + Alt + T`, then type in the following commands:
```

Comment:
Can you copy and paste or does it have to be type? 

If the former, recommend changing wording to _"... then enter the following commands:"_ Then the user can simply copy from the website into terminal and avoid errors. 

If the latter, no change.

Reply:
fixed